.gitignore

```diff
@@ -16,3 +16,5 @@ tmp/*
 coverage_report/
 .coverage
 __pycache__/
+
+.DS_Store
```

Comment:
This should go in your own global gitignore file

Reply:
wooo so  with this global gitignore I don't have to keep writing .DS_Store??  I followed the instruction, I hope it worked...